.packit.yaml

```diff
@@ -69,3 +69,19 @@ jobs:
     targets: *pr_copr_targets
     identifier: podman_system_test
     tmt_plan: "/plans/podman_system_test"
+
+  - job: propose_downstream
+    trigger: release
+    update_release: false
```

Comment:
The update release option does not work with `propose_downstream`, but since you use `autorelease` macro downstream, this should be preserved so I think the default behaviour should be fine.

If you want to see the result, you can use [ `packit propose-downstream --local-content` via CLI](https://packit.dev/docs/cli/propose-downstream/) to open the PRs so you don't need to wait for the new release.

Reply:
@lachmanfrantisek ack, i think downstream tasks are hard to do if  `container-selinux.spec` is not in the upstream checkout, correct? Currently, we have `container-selinux.spec.rpkg` upstream along with a:
```
post-upstream-clone:
    - rpkg spec --outdir ./
```

I think it might be easiest for us to just switch from `.spec.rpkg` to `.spec` upstream and have it sync with downstream via packit.